web-client/src/modules/request/index.ts

```diff
@@ -1,10 +1,92 @@
+import {
+  ContainerOutlined,
+  FileAddOutlined,
+  FileDoneOutlined,
+  FileProtectOutlined,
+  FileSearchOutlined,
+  FileSyncOutlined,
+} from '@ant-design/icons';
+import { uniqueId } from 'lodash';
+import React from 'react';
+import Location from 'react-app-location';
 import { Module } from 'src/types/module';
 
+import { MenuItem } from '../../types/menu-item';
 import ContentPage from './pages/ContentPage';
+import { NewRequestLocation } from './pages/routes/NewRequestRoute/constants';
+
+// TODO (remove link id): ask help to find a better way to keep track of ids
+export const menuItems: Array<MenuItem> = [
+  {
+    id: uniqueId(),
+    icon: React.createElement(FileAddOutlined),
```

Comment:
I'm not sure that this is a good way. I'm open for any advice in here.

Reply:
Have you checked this out? https://ant.design/components/icon/

Reply:
Yes, the icons here I passed are exactly the antd icons.

If I use like below:
```javascript
{
  id: uniqueId(),
  icon: FileAddOutlined,
}
```
Then, I have to createElement from that with the same in render method:
```javasript
<Menu.Item key={item.id} {...other}>
   {item.icon && React.createElement(item.icon)}
   {item.title}
</Menu.Item>
```

Also, I've just checked from react-router-dom source code and saw they handled it with the similar way.

```javascript
<Route path={''} component: SomeComponent />
// correspondent block from source code
component
  ? React.createElement(component, props)
  : render
```

Reply:
Ok, thanks for explaining :)

Reply:
I have fixed this issue in a commit on this branch.
I don't think it's a good idea to put this logic in the modules.
It's better to put this logic into the dashboard layout component and thus abstract it away.
The use of `React.createElement` is fine, but it should be in the dashboard layout component itself.
using `React.createElement(x)` is the same as using `<x />` in JSX and in this particular case since it's an object and we aren't using React to render this, you have to use `React.createElement`, however, in the method I used in the latest commit, this is no longer required and am using JSX.